src/main/webapp/WEB-INF/tags/instructor/comments/commentsForStudentsPanel.tag

```diff
@@ -0,0 +1,83 @@
+<%@ tag description="Comments for students" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor/comments" prefix="comments" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="commentsForStudentsTables" type="java.util.Collection" required="true" %>
+
+<div class="panel panel-primary">
+    <div class="panel-heading">
+        <strong>${data.viewingDraft ? 'Comment drafts' : 'Comments for students'}</strong>
+    </div>
+    <div class="panel-body">
+        ${data.viewingDraft ? 'Your comments that are not finished:' : ''}

```

Comment:
I recommend not using any variables from `data` aside those in `PageData` (i.e. don't use variables from `InstructorCommentsPageData`, just `PageData`).
Try to pass in other variables from the jsp / tag that uses this.